src/pages/blog/article.jsx

```
@@ -11,117 +13,71 @@ import {
-<br></br>
-
-<h4><b>References:</b></h4>
-
-<ul>
-  <li><a href="https://www.acog.org/womens-health/faqs/nutrition-during-pregnancy">https://www.acog.org/womens-health/faqs/nutrition-during-pregnancy</a></li>
-  <li><a href="https://www.cdc.gov/ncbddd/folicacid/about.html">https://www.cdc.gov/ncbddd/folicacid/about.html</a></li>
-  <li><a href="https://www.nichd.nih.gov/health/topics/pregnancy/conditioninfo/nutrition">https://www.nichd.nih.gov/health/topics/pregnancy/conditioninfo/nutrition</a></li>
-</ul>
-
+        <div
+                dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)